### PR TITLE
Add -y flag to accept default prompts

### PR DESCRIPTION
**Pull request by trstruth**
I've been trying to run the setup script in a Dockerfile command, and the option prompting is breaking that.  Instead of trying to pipe in answers in a hacky way, I thought I'd try adding an "accept all default prompts" option to the script.  The usage is similar to `apt-get install -y <pkg>`, which prevents apt from prompting whether the user wants to install or not.

No clue if this is something the maintainers are interested in pulling in, but thought a PR couldn't hurt :) 
 I've done some manual testing and it seems the functionality remains unchanged when the `-y` opt isn't supplied.  Feedback and/or change requests welcome!

### CODE REVIEW

File: setup [hunk 1]
```diff
@@ -4,13 +4,35 @@ set -e
...
+ACCEPT_DEFAULTS=0
+
 # Attempts to determine the number of cores in the CPU
 # Source: https://gist.github.com/jj1bdx/5746298
 CPUS=$(getconf _NPROCESSORS_ONLN 2>/dev/null)
 [ -z "$CPUS" ] && CPUS=$(getconf NPROCESSORS_ONLN)
 [ -z "$CPUS" ] && CPUS=$(ksh93 -c 'getconf NPROCESSORS_ONLN')
 [ -z "$CPUS" ] && CPUS=1
 
+while getopts 'y' flag; do
```
Comment (eigenform) [inline setup]: Perhaps let this be `while getopts 'yh' flag` so `-h` is recognized.

**Review by eigenform — COMMENTED** [review]
Hi! I'll totally merge this! Just fix the `-h` flag; I think you need to include it in `getopt`?

File: setup (hunk 1 above)
Comment (trstruth) [reply setup]: good catch!  I pushed a commit to fix it

Comment (eigenform) [reply setup]: Thanks!!

**Pull request merged**